docs/source/ko/_toctree.yml

```diff
@@ -36,8 +36,8 @@
 #    title: (번역중) Build a web browser agent using vision models
   - local: examples/using_different_models
     title: 다양한 모델 사용하기
-#  - local: examples/plan_customization
-#    title: "(번역중) Human-in-the-Loop: Customize agent plan interactively"
+  - local: examples/plan_customization
+    title: Human-in-the-Loop: 사용자와 상호작용하며 에이전트 계획 수정하기
```

Comment:
We need the quotes here to avoid breaking the doc building: https://github.com/huggingface/smolagents/actions/runs/18935709361/job/54061973476
```
yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 40, column 29:
        title: Human-in-the-Loop: 사용자와 상호작용하며 에이전트 계획 수정하기
                                ^
Error: Process completed with exit code 1.
```